src/VersionTool/ProcessRunner.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace VersionTool
+{
+    internal class ProcessRunner
```

Comment:
In fact, this may not be needed anymore. I added this to be able to run `git diff` but as per the comments below. That mechanism of detecting package version changes may not be needed anymore since we pass them in via command line args.